scene/gui/tab_bar.cpp

```diff
@@ -1485,7 +1485,7 @@ void TabBar::_ensure_no_over_offset() {
 }
 
 void TabBar::ensure_tab_visible(int p_idx) {
-	if (!is_inside_tree() || !buttons_visible) {
```

Comment:
You could use something similar to `setting_up` here.

Reply:
I instead call `ensure_tab_visible()` on `NOTIFICATION_ENTER_TREE`.